src/SafeAnchor.js

```diff
@@ -55,8 +58,18 @@ class SafeAnchor extends React.Component {
     }
   }
 
+  handleKeyDown(event) {
+    event.preventDefault();
+    if (event.key === ' ') {
+      this.handleClick(event);
+    }
+  }
+
   render() {
-    const { componentClass: Component, disabled, ...props } = this.props;
+    const { componentClass: Component, disabled, onKeyDown, ...props } = this.props;
+
+    const handleKeyDown = Component === 'a' ?
+      createChainedFunction(this.handleKeyDown, onKeyDown) : onKeyDown;
```

Comment:
oops, sorry, i take that back – i forgot that using `<button>` was handled upstream in `<Button>`. you don't need to check for `a` here